src/scene/vcPOI.cpp

```diff
@@ -776,6 +779,9 @@ void vcPOI::HandleImGui(vcState *pProgramState, size_t *pItemID)
       vcWebFile_OpenBrowser(m_hyperlink);
   }
 
+  if (vcIGSW_InputText(vcString::Get("scenePOILabelDescription"), m_description, sizeof(m_description), ImGuiInputTextFlags_EnterReturnsTrue))
```

Comment:
For future reference, the reason I said `udLengthOf` and not `sizeof` is that `sizeof` will work on every variable. If `m_description` changes to a pointer instead, this code will continue to compile. `udLengthOf` only works on arrays, so this line would fail to compile.

Reply:
sounds good